tools/omfwd.c

```diff
@@ -1284,6 +1313,11 @@ CODESTARTnewActInst
 					"cannot be used with tcp transport -- ignored");
 		}
 	}
+
+	if(pData->address && (pData->protocol == FORW_TCP)) {
+		errmsg.LogError(0, RS_RET_PARAM_ERROR,
```

Comment:
same here re ```errmsg.``` (see line 1258 for current use)

Reply:
Done

Reply:
thx!